### DISCUSSION TITLE
Card ColumnSize gone?

### DISCUSSION THREAD

**Discussion by svenbaus** [Megabit/Blazorise#4741] [Q&A] [answered]
Hello,

I'm currently updating my project references and therefore coming from Blazorise 1.1.6 to 1.2.2 the following sizing doesn't work:

```
<CardDeck Margin="Margin.Is1.OnX">
    <AuthorizeView Roles="admin, Modules.TestdataProvisioning">
        <Card ColumnSize="ColumnSize.Is6">
            <CardImage Source="/img/document-preview.png" @onclick='() => _navigationManager.NavigateTo("/testdataprovisioning/initialization")'></CardImage>
            <CardBody>
                <CardTitle Size="5">@_localizer["Testdataprovisioning"]</CardTitle>
                <CardText>@_localizer["Module providing utilities for testdata and their provisioning."]</CardText>
                <Button Color="Color.Primary" Block Clicked='() => _navigationManager.NavigateTo("/testdataprovisioning/initialization")'>@_localizer["Open module"]</Button>
            </CardBody>
        </Card>
    </AuthorizeView>
</CardDeck>
```

It seems like <Card> doesn't have a ColumnSize anymore. How can I size a Card?

**Comment by stsrki** [comment]
We made some changes to the columns, and one of the changes was that Card ColumnSize API had to be removed. You can try one of the following.

1. Wrap the card with Column

```razor
<Row>
  <Column ColumnSize="ColumnSize.Is6">
    <Card>...</Card>
  </Column>
</Row>
```

2. add class name to the Class parameter

```razor
<Card Class="col-6">
  ...
</Card>
```

**Reply by svenbaus** [reply]
What about the CardDeck? Should I wrap it inside or is the CardDeck replaced by the Row?

**Reply by stsrki** [reply, marked as answer]
🤔I guess if you want to leave CardDeck you will need to go with the second option, `<Card Class="col-6">`.

**Reply by svenbaus** [reply]
Thanks, that works :).
